bigtable-client-core-parent/bigtable-client-core/src/test/java/com/google/cloud/bigtable/grpc/TestCallOptionsFactory.java

```diff
@@ -100,7 +108,8 @@ public void run() {
                 CallOptionsConfig.builder()
                     .setUseTimeout(true)
                     .setShortRpcTimeoutMs((int) TimeUnit.SECONDS.toMillis(100))
-                    .setLongRpcTimeoutMs((int) TimeUnit.SECONDS.toMillis(1000))
+                    .setMutateRpcTimeoutMs((int) TimeUnit.SECONDS.toMillis(1000))
```

Comment:
can we add more tests with just .setMutateRpcTimeoutMs, one with just .setReadRwosRpcTimeoutsMs, and one with no settings?

Reply:
added more tests to cover these scenarios.